public/main.js

```diff
@@ -1 +1,88 @@
-document.querySelector('body').innerHTML = '<h1>タイトル</h1>';
\ No newline at end of file
+import {ButtonComponent} from"./buttonComponent.js";
+import {TitleComponent} from"./titleComponent.js"
+import {getQuiz} from"./getQuiz.js";
+
+const buttonComponent=new ButtonComponent();
+const titleComponent=new TitleComponent();
+titleComponent.setAttribute("text", "");
+const panel=document.querySelector(".panel");
+panel.appendChild(titleComponent);
+panel.appendChild(buttonComponent);
+
+
+let quizNum=0;
+let quizData;
+
+window.onload = function() {
```

Comment:
```suggestion
window.onload = function(){
```

ほかのコードを見る感じ`{}`の前にスペースを入れない主義っぽいので、ここもなくていいと思います。